src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -2054,6 +2054,8 @@ lVbRuntimePlus:
                         Return Platform.AnyCpu32BitPreferred
                     Case "arm"
                         Return Platform.Arm
+                    Case "arm64"
```

Comment:
Since we are changing VB. We should have at least one VB test for this. Even though most of the functionality is in common code.
Perhaps add scenarios to `CompareAllBytesEmitted_Release` - there is a  VB variant of that too.